### PR TITLE
#2535 Fix Object Descriptions

### PR DESCRIPTION
**Pull request by StrawberryMoses**
### Purpose
Checks if Attributes component exists when setting tooltip. Not every object has Attributes, but if it does, it should have priority. Eventually everything (I think?) should have attributes.

### Please make sure you have followed the self checks below before submitting a PR:

- Code is sufficiently commented
- Code is indented with tabs and not spaces
- The PR does not include any unnecessary .meta, .prefab or <b>.unity (scene) changes</b>
- The PR does not bring up any new compile errors
- The PR has been tested in editor
- Any new files are named using PascalCase (to avoid issues on case sensitive file systems)
- Any new / changed components follow the [Component Development Checklist](https://github.com/unitystation/unitystation/wiki/Component-Development-Checklist)
- Any new objects / items follow the [Creating Items and Objects Guide](https://github.com/unitystation/unitystation/wiki/Creating-Items-and-Objects%3A-Now-With-Prefab-Variants) (especially concerning the use of prefab variants)
- The PR has been tested in multiplayer (with 2 clients and late joiners, if applicable)
- The PR has been tested with round restarts.
- The PR has been tested on moving / rotating / rotated-before-joining matrices (if applicable)

### CODE REVIEW

**Comment by chairbender** [comment]
sorry @StrawberryMoses there's a merge conflict in Attributes because of something I just merged

**Comment by StrawberryMoses** [comment]
no worries fix will be up in a minute

File: UnityProject/Assets/Scripts/Items/Attributes.cs [hunk 1]
```diff
@@ -99,13 +99,13 @@ private void SyncArticleDescription(string newDescription)
 	public void OnHoverStart()
 	{
 		UIManager.SetToolTip =
-			initialName.First().ToString().ToUpper() + initialName.Substring(1) + 
-			(String.IsNullOrEmpty(articleDescription) ? "" : $" ({articleDescription})");
+			initialName.First().ToString().ToUpper() + initialName.Substring(1) +
```
Comment (chairbender) [inline UnityProject/Assets/Scripts/Items/Attributes.cs:102]: This causes a "sequence is empty" exception in the logs when mousing over some things, I think when they have empty string for their name? There's something in the shuttle that I can mouse over to cause this.

**Review by chairbender — CHANGES_REQUESTED** [review]

File: UnityProject/Assets/Scripts/Items/Attributes.cs (hunk 1 above)
Comment (chairbender) [reply UnityProject/Assets/Scripts/Items/Attributes.cs:102]: nvm I'll fix this myself since I missed it when I reviewed another PR

**Pull request merged**